src/tcgwars/logic/impl/gen3/RubySapphireNG.groovy

```diff
@@ -597,14 +700,24 @@ public enum RubySapphireNG implements LogicCardInfo {
         pokePower "Energy Trans", {
           text "As often as you like during your turn (before your attack), move a [G] Energy card attached to 1 of your Pokémon to another of your Pokémon. This power can't be used if Sceptile is affected by a Special Condition."
           actionA {
+            checkNoSPC()
+            assert my.all.findAll {it.cards.energyCount(G)>0} : "There are no Pokémon with [G] Energy cards"
```

Comment:
This should probably be either a `find`, `any`, or should store the results of the `findAll` before asserting it so that you don't have to repeat the `findAll` later.

Reply:
Yup, fixing it (migrated card from POP)

```suggestion
            assert my.all.find{it.cards.energyCount(G)>0} : "There are no Pokémon with [G] Energy cards"
```